services/core-api/app/api/notice_of_departure/models/notice_of_departure.py

```diff
@@ -101,20 +125,37 @@ def find_one(cls, __guid, include_documents=False):
         return cls.query.filter_by(nod_guid=__guid, deleted_ind=False).first()
 
     @classmethod
-    def find_all_by_mine_guid(cls, __guid):
-        return cls.query.filter_by(
-            mine_guid=__guid, deleted_ind=False).order_by(cls.create_timestamp.desc()).all()
+    def find_all(cls, mine_guid=None, permit_guid=None, order_by=None, order=None):
 
-    @classmethod
-    def find_all_by_permit_guid(cls, __guid, mine_guid=None):
-        query = cls.query.filter_by(
-            permit_guid=__guid, deleted_ind=False).order_by(cls.create_timestamp.desc())
+        query = cls.query.filter_by(deleted_ind=False)
         if mine_guid:
-            query = cls.query.filter_by(
-                permit_guid=__guid, mine_guid=mine_guid,
-                deleted_ind=False).order_by(cls.create_timestamp.desc())
+            query = query.filter_by(mine_guid=mine_guid)
+        if permit_guid:
+            query = query.filter_by(permit_guid=permit_guid)
+
+        if (order_by):
+            if (order == 'asc'):
+                query.order_by(asc(text(order_by.value)))
+            else:
+                query.order_by(desc(text(order_by.value)))
+
         return query.all()
 
+    # @classmethod
```

Comment:
Are you waiting to get rid of these until we've verified that everything is wired up properly?